modules/navier_stokes/doc/content/modules/navier_stokes/pinsfv.md

```diff
@@ -0,0 +1,146 @@
+
+- free slip walls
+  !listing modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/2d-rc.i block=FVBCs/free-slip-u
+
+- symmetry axis. This symmetry condition should also be indicated for the pressure variable.
+  !listing modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/2d-rc.i block=FVBCs/symmetry-u
+  !listing modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/2d-rc.i block=FVBCs/symmetry-p
+
+- inlet mass flux
+  !listing modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/2d-rc.i block=FVBCs/inlet-u
+
+- momentum advection outflow (only for a mean-pressure approach, equivalent to executing the momentum advection kernel on the boundary)
+  !listing modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/2d-rc.i block=FVBCs/outlet-u
+
+
+The pressure boundary condition is usually only set at the outlet:
+!listing modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/2d-rc.i block=FVBCs/outlet-p
+
+For a mean-pressure approach, usually for cavity problems, the user may specify a mass advection boundary condition. This
+is equivalent to executing the mass advection kernel on boundaries.
+!listing modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/2d-rc.i block=FVBCs/outlet-p-novalue
+
+## Example inputs : heated straight channel
+
+This channel features a porous media with temperature exchanged by convection between the fluid
+and the solid phase. The fluid temperature obeys the porous media Navier Stokes energy equation with
+both advection and diffusion of the fluid energy. The inlet boundary condition is set to a
+`FVNeumannBC` to specify an incoming flux. An alternative would be to use a `FVDirichletBC` to specify
+an inlet temperature, but depending on the numerical scheme for the computation of advected quantities
+on faces, this is not as exact.
+
+The solid temperature is by default constant in this input file, however by activating the relevant non-linear `Variable`,
+`FVKernels` and `FVBCs`, this input also allows solving for the solid phase temperature.
+
+!listing modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/2d-rc-heated.i
+
+## Example inputs : straight channel with a porosity change
+
+This channel features a change in porosity between 1 and 0.5 with either a sharp discontinuity or
+a continuous variation between the two regions, depending on the porosity initial condition (`ICs` block).
+
+Both Rhie Chow interpolation and the momentum equation feature a pressure gradient, and the superficial momentum
+advection and diffusion terms feature a porosity or inverse porosity gradient. These are ill-defined near the
+discontinuity. To deal with this issue, the discontinuous case uses a flux-based formulation of the pressure gradient,
+`PINSFVMomentumPressureFlux` and Rhie Chow interpolation reverts to average interpolation on the faces near the discontinuity.
```

Comment:
Does the Rhie Chow interpolation still revert to average interpolation near the discontinuity?

Reply:
I took out that in the latest commit.

Reply:
so oscillations in the no-pressure-gradient cases are back

Reply:
Got it. Yea guessing we'll just have to `FVInterfaceKernel` if we really want to suppress that.